CefSharp.Core/Internals/Serialization/V8Serialization.cpp

```diff
@@ -102,6 +102,47 @@ namespace CefSharp
                 {
                     SetCefTime(list, index, ConvertDateTimeToCefTime(safe_cast<DateTime>(obj)));
                 }
+                // Serialize enum to sbyte, short, int, long, byte, ushort, uint, ulong (check type of enum)
+                else if (type->IsEnum)
+                {
+                    auto subType = System::Enum::GetUnderlyingType(type);
+                    if (subType == SByte::typeid)
```

Comment:
Can we change this to a switch and group the similar types together?

Reply:
Of course, please fix my code as you wish :)